src/commons/assessmentWorkspace/AssessmentWorkspace.tsx

```diff
@@ -184,11 +185,11 @@ const AssessmentWorkspace: React.FC<AssessmentWorkspaceProps> = props => {
     };
   }, [dispatch]);
 
-  useEffect(() => {
-    // TODO: Hardcoded to make use of the first editor tab. Refactoring is needed for this workspace to enable Folder mode.
-    handleEditorValueChange(0, '');
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  // useEffect(() => {
+  //   // TODO: Hardcoded to make use of the first editor tab. Refactoring is needed for this workspace to enable Folder mode.
+  //   handleEditorValueChange(0, '');
+  //   // eslint-disable-next-line react-hooks/exhaustive-deps
+  // }, []);
```

Comment:
Will this break anything with assessments? Have you tested locally?

Reply:
From my local testing, there's no unintended behavior. Assessments still loading as per normal.

handleEditorValueChange is ultimately called with the editor contents due to the useEffect [here](https://github.com/source-academy/frontend/blob/4e43c66fd338b9bd2870651853fb3fd177e03a22/src/commons/assessmentWorkspace/AssessmentWorkspace.tsx#L252).

Reply:
Let's just delete and clean up all this unused code then